docs/changes/latest.inc

```diff
@@ -35,6 +35,8 @@ Enhancements
 - Add :class:`junifer.markers.EdgeCentricFCParcels` and :class:`junifer.markers.EdgeCentricFCSpheres`
   (:gh:`64` by `Leonard Sasse`_).
 
+- Add seven complexity markers (:gh:`61` by `Amir Omidvarnia`_, `Leonard Sasse`_).
```

Comment:
Would be good to have the references to the classes here.

Reply:
The references to all features were added to latest.inc.